pkg/io/storage/storage.go

```diff
@@ -0,0 +1,90 @@
+/*
+Package storage provides functions for storing and retrieving data.
+from json file
```

Comment:
I could suggest that this package in overall is not about json file but about a way to store data somehow (in file by xml or json, in memory, or if to imaging, in some NoSQL database, who knows). So it will be more clear to leave only general description of a package and remove implementation-specific details.